pkg/trtl/metrics.go

```diff
@@ -0,0 +1,141 @@
+package trtl
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/rs/zerolog/log"
+)
+
+var (
+	pmPuts  *prometheus.CounterVec // count of trtl Puts per namespace
+	pmGets  *prometheus.CounterVec // count of trtl Gets per namespace
+	pmDels  *prometheus.CounterVec // count of trtl Deletes per namespace
+	pmIters *prometheus.CounterVec // count of trtl Iters per namespace
+	// pmObjects    *prometheus.CounterVec   // count of objects being managed by trtl, by namespace
+	// pmTombstones *prometheus.CounterVec   // count of tombstones per namespace; increases on delete, decrease on overwrite of tombstone
```

Comment:
These might not be possible yet - see my notes inline; though perhaps I missed some other opportunity that you'll see!

Reply:
Yes, I see exactly what you mean - excellent notes and comments below! The only thing I can think of is a go routine that kicks off every 10 minutes or so and counts all the objects and tombstones in the database. Otherwise there are some changes to Honu we could make to make the metrics happen correctly. Either way, I think this is probably a follow on task we could create a story for?